test/unit/gateways/moneris_us_test.rb

```diff
@@ -22,6 +22,21 @@ def test_default_options
     assert_equal "qatoken", @gateway.options[:password]
   end
 
+  def test_successful_verify
+    @gateway.expects(:ssl_post).twice.returns(successful_purchase_response)
```

Comment:
I would have expected to see successful_authorize_response here since under the covers, it's doing an authorize rather than a purchase.  Perhaps something along the lines of:

https://github.com/Shopify/active_merchant/blob/master/test/unit/gateways/bridge_pay_test.rb#L105-L127

In Bridgepay's case, it's actually doing a void so in Moneris US, perhaps it could stub out `successful_authorize` followed by a successful capture of $0, since that seems to be the bizarro way to void an auth for Moneris US.